<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="marsis">
       <property name="title">
        <string>MARSIS</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Simulations</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="radioMarsisDataHttp">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMarsisDataHttp"/>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Disk source</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>HTTP source</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioMarsisSimHttp">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMarsisSimHttp"/>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="radioMarsisSimDisk">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_2</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMarsisSimDisk"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolMarsisSimDisk">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Data</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="radioMarsisDataDisk">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineMarsisDataDisk"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolMarsisDataDisk">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="sharad">
       <property name="title">
        <string>SHARAD</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="2" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>HTTP source</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Simulations</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Data</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Disk source</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QRadioButton" name="radioSharadDataDisk">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_3</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineSharadDataDisk"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolSharadDataDisk">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QRadioButton" name="radioSharadSimDisk">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_4</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineSharadSimDisk"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolSharadSimDisk">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QRadioButton" name="radioSharadDataHttp">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_3</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineSharadDataHttp"/>
            </item>
           </layout>
          </item>
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QRadioButton" name="radioSharadSimHttp">
              <property name="text">
               <string/>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroup_4</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineSharadSimHttp"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="misc">
       <property name="minimumSize">
        <size>
         <width>657</width>
         <height>190</height>
        </size>
       </property>
       <property name="title">
        <string>Misc</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Cache dir</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineCacheDir"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolCacheDir">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item alignment="Qt::AlignRight|Qt::AlignBottom">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>839</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolCacheDir</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>toolCacheClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>815</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolMarsisDataDisk</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>toolMarsisDataClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>521</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolMarsisSimDisk</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>toolMarsisSimClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>821</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolSharadDataDisk</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>toolSharadDataClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolSharadSimDisk</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>toolSharadSimClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toolCacheClicked()</slot>
  <slot>toolMarsisDataClicked()</slot>
  <slot>toolMarsisSimClicked()</slot>
  <slot>toolSharadDataClicked()</slot>
  <slot>toolSharadSimClicked()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
